src/app/login/login.component.html

```diff
@@ -1,16 +1,9 @@
-<p>login works!</p>
-<mat-card class="card-row">
-  <button color="primary" mat-raised-button (click)="goToTutorial()">
-    Visit Tutorial Page
-  </button>
-</mat-card>
-<mat-card class="card-row">
-  <button color="warn" mat-raised-button (click)="goToReference()">
-    Visit Reference Page
-  </button>
-</mat-card>
-<mat-card class="card-row">
-  <button color="accent" mat-raised-button (click)="goToProblem()">
-    Visit Problem Page
-  </button>
-</mat-card>
+
+<div class="wrapper login-card">
```

Comment:
I thought it was an angular practice to use `:host` instead of wrapping `div`s?

Reply:
haven't used :host like that before, also not seen this usage. Generally we use it to target the style of parent components. Sometimes we want parent component to reflect some style dictated by injected components. But at the same time we don't want the parent to always have that style (example parent component maybe getting re-used). In that case we add :host in child and the particular parent component of a child will reflect the style.

Reply:
That's not the usage that I've read. But you're the angular expert. I will read more about it later.